impeller/renderer/compute_pass.cc

```diff
@@ -47,6 +47,11 @@ bool ComputePass::AddCommand(ComputeCommand command) {
 }
 
 bool ComputePass::EncodeCommands() const {
+  if (grid_size_.IsEmpty() || thread_group_size_.IsEmpty()) {
```

Comment:
Ditto about the validation logs.

Reply:
Same comment - I think making this a fatal error is too strong.